lib/instance.js

```diff
@@ -128,7 +130,9 @@ export const entries = function* (node) {
   }
 
   for (const property of node.children) {
-    yield property.children;
+    if (node.children.length === 2){
```

Comment:
The bug causing the test failures is here.

```suggestion
    if (property.children.length === 2) {
```